hyperspace/parachain/src/chain.rs

```diff
@@ -132,13 +137,24 @@ where
 				.encoded()
 				.to_vec()
 		};
-		let dispatch_info = TransactionPaymentApiClient::<
-			sp_core::H256,
-			RuntimeDispatchInfo<u128, u64>,
-		>::query_info(&*self.para_ws_client, extrinsic.into(), None)
-		.await
-		.map_err(|e| Error::from(format!("Rpc Error From Estimating weight {:?}", e)))?;
-		Ok(dispatch_info.weight)
+		let dispatch_info =
+			TransactionPaymentApiClient::<sp_core::H256, RuntimeDispatchInfo>::query_info(
+				&*self.para_ws_client,
+				extrinsic.into(),
+				None,
+			)
+			.await
+			.map_err(|e| Error::from(format!("Rpc Error From Estimating weight {:?}", e)))?;
+
+		#[cfg(feature = "dali")]
```

Comment:
it will be the same for Picasso and composable right?

Reply:
if so, then we can just add it right now

Reply:
same

Reply:
just used already existing feature.